### ISSUE TITLE
US data files missing

### ISSUE THREAD

**Issue by ParkerR-ThompsonDryers** [CSSEGISandData/COVID-19#1428]
I see mention in previous issues that data for the US will now be updated in separate files, and will be delineated by state and county. I cannot find those files anywhere.

**Comment by MEM-TEK** [comment]
Look in the csse_covid_19_daily_reports folder.

**Comment by johncdavis200** [comment]
When are the time_series_covid19_confirmed_US.csv and time_series_covid19_deaths_US.csv files going to be delivered? and are they going to be in the same location as the global data?

**Comment by ParkerRobb** [comment]
> Look in the csse_covid_19_daily_reports folder.

@MEM-TEK I'm talking about this paragraph from #1250 

> Three new time series tables will be added for the US. The first two will be the confirmed cases and deaths, reported at the county level. The third, number of tests conducted, will be reported at the state level. These new tables will be named time_series_covid19_confirmed_US.csv, time_series_covid19_deaths_US.csv, time_series_covid19_testing_US.csv, respectively.

These files are nowhere to be found.

**Comment by jdtsmith** [comment]
Yes, those files are critically needed for tracking state-by-state growth trajectory.

**Comment by theronrr** [comment]
https://github.com/ulklc/covid19-timeseries/blob/master/report/country/US.csv

**Comment by kondis** [comment]
https://github.com/CSSEGISandData/COVID-19/issues/1426

**Comment by k2xl** [comment]
Okay, an update for anyone reading this - all of these referenced issues don't really answer the original question.

Here's the status from what I can tell:

1. `time_series_covid19_confirmed_US.csv` and `time_series_covid19_deaths_US` doesn't exist right now in the repo
2. Yes, the US state/county data exists in the daily reports, but it doesn't seem like it goes back to the beginning. So #1 would be the ideal fit because everyone could just join that with the original CSV file
3. Recovered data is no longer around because the data isn't reliable. Yes that means you can't plot the recovered data. Yes that means you can't "infer" active cases by subtracting recovered from confirmed (I had to remove it from my website https://covidtrend.com)

**Comment by jyelon** [comment]
Guys, this is a real issue.  The US timeseries data is entirely inaccessible.  It's valuable data! Please don't ignore this.
